production/scrapers/iowa_population.R

```diff
@@ -0,0 +1,86 @@
+source("./R/generic_scraper.R")
+source("./R/utilities.R")
+
+iowa_population_pull <- function(x){
+    xml2::read_html(x)
+}
+
+iowa_population_restruct <- function(x){
+    x %>%
+        rvest::html_nodes("table") %>%
+        .[[1]] %>%
+        rvest::html_table(fill = TRUE)
+}
+
+iowa_population_extract <- function(x){
+    expected_names <- c(
+        "Institution", "Current Count", "Capacity", "Medical/Segregation")
+    
+    check_names(x, expected_names)
+    
+    x %>% 
+        select(Name = Institution, 
+               Residents.Population = `Current Count`) %>% 
+        filter(!Name %in% c("INSTITUTIONAL TOTALS", "% overcrowded by")) %>% 
+        # Rename and aggregate facility names to match COVID data 
+        mutate(Name = case_when(
+            # Report Live-Out separately, rename facility to denote this 
+            Name == "Minimum Live-Out" & lag(Name) == "Mitchellville" ~ "Mitchellville Minimum Live-Out", 
+            # Report Psychiatric separately, rename facility to denote this 
+            Name == "Forensic Psychiatric Hospital" & lag(Name == "Oakdale") ~ "Oakdale Forensic Psychiatric Hospital", 
```

Comment:
This is super tiny, but does it matter that the way you're using parentheses here differs from the above and below cases? E.g,    lag(Name == "Oakdale")    versus   lag(Name) == "Mitchellville". Only asking because I have been stung by parentheses leading to strange behavior in the past!

Reply:
Whoops good catch! Fixed the parentheses (and genuinely shocked that R had the same behavior regardless, too forgiving of a language imo). Thanks!